### ISSUE TITLE
Clear all dropzone

### ISSUE THREAD

**Issue by buzzer83** [Postlagerkarte/blazor-dragdrop#23]
Hi, I would like to clear all Draggable element in a Dropzone, is it possible? I read #6 and I understand to delete a single element, and I can use this for all, but I would like to find a better way. TY

**Comment by Postlagerkarte** [comment]
Hi @buzzer83 ,

currently there is no such method but it really makes sense to have one.  You want to provide a PR for it?

**Comment by buzzer83** [comment]
Hi Postlagerkarte, thank you for developing the ClearDropzone method. I have tested your method but in my view it does not clean the dropzone. I wrote the file example.razor to explain what isn't working.

I'll list some use cases:
1. if I select any of the three choices the first time (1 to 2, 1 to 3, 1 to 4) I see correctly the lists of numbers.
2. if I select "1 to 2" and after "1 to 4" I view only the lasts 2 number
3. if I select "1 to 2" and after "1 to 3" I view only the last number
From the tests it can be seen that only the number of extra entries from the last selection are displayed.

If I have not explained myself well I can make you a short video.
TY
[example.blazor.txt](https://github.com/Postlagerkarte/blazor-dragdrop/files/4809386/example.blazor.txt)
